flink-connectors/flink-hbase/src/test/java/org/apache/flink/addons/hbase/HBaseLookupSourceITCase.java

```diff
@@ -200,4 +203,44 @@ public void testHBaseLookupFunction() throws Exception {
 
 		StreamITCase.compareWithList(expected);
 	}
+
+	@Test
+	public void testHBaseLookupTableSource() throws Exception {
+		EnvironmentSettings settings = EnvironmentSettings.newInstance()
+			// only blink planner support lookup table source
+			.useBlinkPlanner().inStreamingMode().build();
```

Comment:
We should use parameterized test to both cover blink-planner and flink-planner just like the way in `HBaseSinkITCase`, and remove the `testHBaseLookupFunction` test.